#!/usr/bin/env python
PACKAGE = "slam_estimator"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("mapping_status", int_t, 0, "0 pause, 1 map, 2 clear map", 0, 0, 2)
gen.add("window_to_map_tracking", bool_t, 0, "Enables consistent mapping", False)

exit(gen.generate(PACKAGE, "slam_estimator", "BlaserSLAM"))